training/src/app/joke/joke.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="body">
+    <div class="rectangle-2">
```

Comment:
dont think this is a good class name. where is rectangle-1? 
rectangle doesnt mean something for someone reading this code.
a better name for example would be joke-container, or joke-wrapper